src/database/models/ubi/manager.ts

```diff
@@ -15,6 +15,11 @@ export interface ManagerAttributes {
 
     user?: User;
     community?: CommunityAttributes;
+    beneficiaryRegistry?: [
+        {
+            address: string;
+        }
+    ];
```

Comment:
I know you are querying only the address on the only use case now, but always use the complete interface (https://github.com/impactMarket/api/blob/master/src/interfaces/ubi/ubiBeneficiaryRegistry.ts#L8) even if you don't query it all.

Reply:
done!